packages/components/src/ButtonDismiss/ButtonDismiss.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styles from "./ButtonDismiss.css";
+import { Button } from "../Button";
+
+interface ButtonDismissProps {
+  onClick?(): void;
```

Comment:
Not sure if I am TypeScripting this portion correctly.... plmk!

What I want to express is the props is expecting a `Function` for the `onClick` prop

Reply:
yep, if you look at https://atlantis.getjobber.com/guides/how-to-create-a-basic-react-component#interface (might have to scroll to the interface section) it shows what is happening here.

You are saying `onClick` accepts a function and will not return anything back.